### PR TITLE
Improve upgrade documentation to take into account pickled versions o…

### PR DESCRIPTION
**Pull request by mbasset**
## Tracker Ticket

https://quandl.atlassian.net/browse/AP-1678

## Description

Add additional documentation for loading the pickled version of files.

## Is there a migration? 

* No

## Note to QA

Test that works with old code.

## Dependency

* None

### CODE REVIEW

File: 2_SERIES_UPGRADE.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ simply change this too:
 import quandl as Quandl
 ```
 
+Additionally if you were relying on a saved version of your `api key` _(authtoken)_ in your scripts please note that this functionality has been dropped from the existing package. To continue to load your api key from the filesystem we recommend you manually load the key and set it via the api config `quandl.ApiConfig.api_key`. Some example code that will do this is:
```
Comment (ccleung) [inline 2_SERIES_UPGRADE.md]: "Manually load" is vague here, maybe we can drop it? To mean that means to actually open up the file in an editor to find my api key, and then copy and paste it into my code?

Also user's might know actually know where the file is, maybe we should reference the old README? https://github.com/quandl/quandl-python#usage

**Comment by ccleung** [comment]
one small comment

File: 2_SERIES_UPGRADE.md (hunk 1 above)
Comment (mbasset) [inline 2_SERIES_UPGRADE.md]: Will try to improve the documentation. However if they do not know where the file is they probably will not have used this feature so will indicate safe to ignore.

**Comment by ccleung** [comment]
:+1:

**Pull request merged**